css/style.css

```diff
@@ -0,0 +1,13 @@
+.answer-choices label {
+    margin-left: 0 !important;
```

Comment:
Consider whether you _need_ !important here, or could you change the specificity. !important had its place, but often leads to a heap of them